pkg/controller/replicaset/replica_set.go

```diff
@@ -215,15 +215,11 @@ func (rsc *ReplicaSetController) getReplicaSetsWithSameController(rs *apps.Repli
 	}
 
 	if klog.V(2).Enabled() {
-		var related string
-		if len(relatedRSs) > 0 {
-			var relatedNames []string
-			for _, r := range relatedRSs {
-				relatedNames = append(relatedNames, r.Name)
-			}
-			related = ": " + strings.Join(relatedNames, ", ")
+		var relatedNames []string
+		for _, r := range relatedRSs {
+			relatedNames = append(relatedNames, r.Name)
 		}
-		klog.Infof("Found %d related %vs for %v %s/%s%s", len(relatedRSs), rsc.Kind, rsc.Kind, rs.Namespace, rs.Name, related)
+		klog.InfoS("Found related ReplicaSets", "replicaset", klog.KObj(rs), "related_replicasets", strings.Join(relatedNames, ","))
```

Comment:
```suggestion
		klog.InfoS("Found related ReplicaSets", "replicaSet", klog.KObj(rs), "related_replicasets", strings.Join(relatedNames, ","))
```

Reply:
if we don't join strings, logs become like this. is it desirable? 
```
I0529 21:54:58.878288   19337 replica_set.go:222] "Found related ReplicaSets" replicaSet="default/rs1" relatedReplicaSets=[]string{"rs1", "rs2"}
```